parts/linux/cloud-init/artifacts/components.json

```diff
@@ -88,7 +104,8 @@
       "multiArchVersionsV2": [
         {
           "renovateTag": "registry=https://mcr.microsoft.com, name=containernetworking/azure-cns",
-          "latestVersion": "v1.4.52",
+          "latestVersion": "v1.4.56",
+          "previousLatestVersion": "v1.4.52",
```

Comment:
@rbtr @ankit-acr Is v1.4.52 the latest previous version published to AKS ? Or v1.5.54?